openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,20 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class is_usda_disabled(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Indicates whether person is disabled per USDA definition, see https://www.law.cornell.edu/uscode/text/7/2014"
```

Comment:
The reference seems to apply more to how the field applies to SNAP.

In the future you can use the `reference = "url"` for that, though I don't think we need it here.
```suggestion
    documentation = "Disabled according to USDA criteria"
```